website/components/info.tsx

```diff
@@ -106,99 +105,109 @@ export default function Info() {
-!..YBBBBBBBBBBBBBBb!!||||||||!iof68BBBBBBRPFT?!::   \`.
-!....YBBBBBBBBBBBBBBbaaitf68BBBBBBRPFT?!:::::::::     \`.
-!......YBBBBBBBBBBBBBBBBBBBRPFT?!::::::;:!^"\`;:::       \`.
-!........YBBBBBBBBBBRPFT?!::::::::::^''...::::::;         iBBbo.
-\`..........YBRPFT?!::::::::::::::::::::::::;iof68bo.      WBBBBbo.
-  \`..........:::::::::::::::::::::::;iof688888888888b.     \`YBBBP^'
-    \`........::::::::::::::::;iof688888888888888888888b.     \`
-      \`......:::::::::;iof688888888888888888888888888888b.
-        \`....:::;iof6888888888888888888888888888888899fT!
-          \`..::!8888888888888888888888888888899fT|!^"'
-            \`' !!988888888888888888888899fT|!^"'
-                \`!!8888888888888888899fT|!^"'
-                  \`!988888888899fT|!^"'
-                    \`!9899fT|!^"'
-                      \`!^"'
-		`}</pre>
-        <div className="col-span-1 flex w-min flex-col items-center justify-start gap-y-10 pt-20 text-xl">
-          <button
-            className={component === "Linux" ? "font-medium" : "opacity-75"}
-            onClick={() => setComponent("Linux")}
-          >
-            Linux
-          </button>
-          <button
-            className={component === "Services" ? "font-medium" : "opacity-75"}
-            onClick={() => setComponent("Services")}
-          >
-            Services
-          </button>
-          <button
-            className={component === "Community" ? "font-medium" : "opacity-75"}
-            onClick={() => setComponent("Community")}
-          >
-            Community
-          </button>
+      <div className="w-full max-w-5xl rounded-lg overflow-hidden shadow-lg bg-gradient-to-b from-[#1a1a2e] to-[#16213e]">
+        <div className="flex flex-wrap border-b border-violet-950">
+          {["Specs", "Linux", "Services", "Community"].map((tab) => (
+            <button
+              key={tab}
+              className={`px-4 py-2 text-sm sm:text-base sm:px-6 sm:py-3 ${
+                component === tab
+                  ? "bg-gray-800 bg-opacity-50 text-HCPurpleText"
+                  : "bg-gray-900 bg-opacity-30 text-white hover:bg-gray-800 hover:bg-opacity-40"
+              } transition-colors flex-grow`}
+              onClick={() => setComponent(tab as "Specs" | "Linux" | "Services" | "Community")}
+            >
+              {tab}
+            </button>
+          ))}
         </div>
-        <div className="col-span-3 h-min rounded-lg bg-gray-900 p-5">
-          {text}
+        <div className="p-4 sm:p-6 h-[300px] sm:h-[400px] overflow-y-auto scrollbar-thin scrollbar-thumb-gray-600 scrollbar-track-gray-900">
+          <AnimatePresence mode="wait">
+            <motion.div
+              key={component}
+              initial={{ opacity: 0, y: 20 }}
+              animate={{ opacity: 1, y: 0 }}
+              exit={{ opacity: 0, y: -20 }}
+              transition={{ duration: 0.3 }}
+            >
+              {component === "Specs" ? (
```

Comment:
I would include a link to the Hetzner page on the server and include the server location

Reply:
done!